### DISCUSSION TITLE
Units for floor plans... seems to be inaccurate in the template? Need a way to scale diagram

### DISCUSSION THREAD

**Discussion by jroachgit** [jgraph/drawio#4432] [Q&A]
Hi all,
Not sure if this is just me or my settings.
I created a new document:
- New -> template -> "floor" -> selected
![image](https://github.com/jgraph/drawio/assets/35711815/51c1c4ed-fee7-45b5-9058-40e3646ffabb)

Then,
- View -> units -> "mm"

Then, I selected a table on the floor plan, and it is: 40mm x 35mm
![image](https://github.com/jgraph/drawio/assets/35711815/cb8afbb0-7b4d-4c08-9328-f59872cbfe32)
If I now select "m", then it changes to 0.040m, as I would except for unit conversion, but 40mm x 35mm is still a bit small for a table.

I tried making the floor plan bigger, but there didn't seem to be a way to make everything magnitudes bigger.
I think this is because the size of the items on the page relates to the actual size of the canvas, i.e. the size of the page. And floor plans are scaled down, they aren't 1:1 of the print size, if that makes sense?

It seems that this was opened as an issue: https://github.com/jgraph/drawio/issues/2333 but went stale. I think its required if you want to do floor plans.

**Comment by Lets-a-go** [comment]
This is true. I have floorplans in Visio documents. The content is not large, but the scale of the page is 1/24, so a foot is 0.5inches on the page. draw.io simply cannot load the document, either online or on the web, sadly, and it cannot load the stencil either, either online or on the web, since the stencil items have similar scale. To make matters worse, almost none of the electronic items have equivalents in draw.io, meaning I am pretty much stuck with an old version of Visio on Windows.
